pkgs/development/python-modules/stytra/default.nix

```diff
@@ -0,0 +1,82 @@
+{ lib, pkgs, buildPythonPackage, fetchPypi, isPy3k, callPackage
+, opencv3
+, pyqt5
+, pyqtgraph
+, numpy
+, scipy
+, numba
+, pandas
+, tables
+, git
+, ffmpeg
+, scikitimage
+, matplotlib
+, qdarkstyle
+, GitPython
+, anytree
+, pims
+, imageio
+, imageio-ffmpeg
+, av
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "stytra";
+  version = "0.8.26";
+  format = "wheel";
+  disabled = !isPy3k;
+
+  arrayqueues = callPackage ./arrayqueues.nix {};
+  colorspacious = callPackage ./colorspacious.nix {};
+  qimage2ndarray = callPackage ./qimage2ndarray.nix {};
+  flammkuchen = callPackage ./flammkuchen.nix {};
+  lightparam = callPackage ./lightparam.nix {};
```

Comment:
please add these as normal packages to python-modules, You should should have a commit for each one

Reply:
done!